app/src/main/java/io/github/droidkaigi/confsched2017/view/transition/LiftOff.java

```diff
@@ -0,0 +1,65 @@
+package io.github.droidkaigi.confsched2017.view.transition;
+
+
+import android.animation.Animator;
+import android.animation.ObjectAnimator;
+import android.content.Context;
+import android.content.res.TypedArray;
+import android.transition.Transition;
+import android.transition.TransitionValues;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.ViewGroup;
+
+import io.github.droidkaigi.confsched2017.R;
+
+
+/**
+ * A transition that animates the elevation of a View from a given value down to zero.
+ * <p>
+ * Useful for creating parent↔child navigation transitions
+ * (https://www.google.com/design/spec/patterns/navigational-transitions.html#navigational-transitions-parent-to-child)
+ * when combined with a {@link android.transition.ChangeBounds} on a shared element.
+ */
+public class LiftOff extends Transition {
```

Comment:
**[Android Lint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Error: Class requires API level 19 (current min is 16): android.transition.Transition [NewApi]